packages/terra-file-path/CHANGELOG.md

```diff
@@ -2,4 +2,6 @@
 
 ## Unreleased
 
+## 0.3.0 - (October 17, 2023)
+
```

Comment:
It's odd that the entries for 0.1 & 0.2 are missing but not sure if it's worth correcting.

Reply:
Yeah I wasn't sure why but the previous entries were removed here: https://github.com/cerner/terra-framework/pull/1825/files#diff-877f2e53487931f7411ca4d8207fdc643647e2c6f97f8d5b68b821c70add4302R5